src/main/java/io/tus/java/client/TusClient.java

```diff
@@ -306,13 +319,14 @@ public TusUploader resumeOrCreateUpload(@NotNull TusUpload upload) throws Protoc
     public void prepareConnection(@NotNull HttpURLConnection connection) {
         // Only follow redirects, if the POST methods is preserved. If http.strictPostRedirect is
         // disabled, a POST request will be transformed into a GET request which is not wanted by us.
-        // See: http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/7u40-b43/sun/net/www/protocol/http/HttpURLConnection.java#2372
+        // todo: Is this link necessary
+        // See:http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/7u40-b43/sun/net/www/protocol/http/HttpURLConnection.java#2372
```

Comment:
This link is way to long for the linter and a line breaker kills the link. 
I could escape this for the linter with two escaping comments, but is this link necessary?

Reply:
The link is not available anymore but please replace it with https://github.com/openjdk/jdk/blob/jdk7-b43/jdk/src/share/classes/sun/net/www/protocol/http/HttpURLConnection.java#L2020-L2035. It is a valuable resource in the future.